src/common/session/session-entity.js

```diff
@@ -140,7 +141,7 @@ export class SessionEntity {
 
     // The fact that the session is "new" or pre-existing is used in some places in the agent.  Session Replay and Trace
     // can use this info to inform whether to trust a new sampling decision vs continue a previous tracking effort.
-    if (this.isNew === undefined) this.isNew = !Object.keys(initialRead).length
+    this.isNew = !Object.keys(initialRead).length
```

Comment:
This is going to break `isNew` since it uses the undefined check in tandem with line 229 in the `reset()` call

Reply:
I removed this bc I think it's extraneous precisely because `reset` will erase the `isNew` property for the `session` manager. Whenever `setup` is ran, `isNew` should be undefined anyways _and_ `isNew` should be set anew in tandem with `initialRead` state.